### PR TITLE
Check if param flag values are defined, not truthiness

### PR DESCRIPTION
**Pull request by zjleblanc**
Addresses #365 by checking if module parameters which represent flags are defined, preventing evaluation of their truthyness. Current behavior will ignore an incoming value of false and default back to true because of the conditional logic:

Example:
`module.params["enabled"] = False`
`if module.params.get("enabled")` evaluates to False, but the intent is to determine if it has been defined, so we want this to evaluate to True.

This PR addresses the same issue for `swap_single_source` param.

### CODE REVIEW

**Comment by Alex-Izquierdo** [comment]
Hi @zjleblanc 
Sorry for the late response, I was on holidays. 
I've checking it deeply and your PR already addresses all the issues, we don't need more changes. The only thing is that we should test this specific issue. Would you mind to add a new task in https://github.com/ansible/event-driven-ansible/blob/main/tests/integration/targets/activation/tasks/main.yml so we can verify that `enabled` param is correctly created? 

Otherwise I can recreate the PR with the test. 

Thank you very much.

**Comment by zjleblanc** [comment]
@Alex-Izquierdo added an assert based on the existing integration tasks - will check back to make sure it passes.

File: tests/integration/targets/activation/tasks/main.yml [hunk 1]
```diff
@@ -404,6 +411,11 @@
         that:
           - event_stream_name in (source_mappings_dict | map(attribute='event_stream_name') | list)
 
+    - name: Assert that swap single source flag is disabled
+      ansible.builtin.assert:
+        that:
+          - not _result_activation_info.activations[0].swap_single_source
```
Comment (Alex-Izquierdo) [inline tests/integration/targets/activation/tasks/main.yml:417]: We don't need this assertion, it is a feature of the collection for configuring event streams. not a field in the backend

Comment (Alex-Izquierdo) [inline tests/integration/targets/activation/tasks/main.yml:417]: We don't need this assertion, it is a feature of the collection for configuring event streams. not a field in the backend

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]

File: tests/integration/targets/activation/tasks/main.yml (hunk 1 above)
Comment (zjleblanc) [reply tests/integration/targets/activation/tasks/main.yml]: got it, removed this assertion 👍

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM, thank you very much for your contribution @zjleblanc

**Review by Dostonbek1 — APPROVED** [review]

**Pull request merged**